### ISSUE TITLE
Migration failed to run update-platform-server-exports from @nrwl/jest. Rule is not a function. This workspace is NOT up to date! 

### ISSUE THREAD

**Issue by dmitri678** [nrwl/nx#16161]
### Current Behavior

Am updating nx from 14.8 to 15.9.2 using nx migrate latest. 

Run "nx migrate --run-migrations" and fails on update-platform-server-exports. Within my migrations.json file the related migration is defined here:

    {
      "cli": "ng",
      "version": "15.2.0-beta.0",
      "requires": {
        "@angular/core": ">=15.0.0"
      },
      "description": "Remove exported `@angular/platform-server` `renderModule` method. The `renderModule` method is now exported by the Angular CLI.",
      "factory": "./src/migrations/update-15-2-0/remove-platform-server-exports",
      "package": "@nrwl/angular",
      "name": "update-platform-server-exports"
    },

Below is detailed error and version info.

>  NX   Failed to run update-platform-server-exports from @nrwl/angular. This workspace is NOT up to date!

 >  NX   rule is not a function

TypeError: rule is not a function
    at callRuleAsync (C:\Users\z0043rkh\Desktop\CP-11-NRWL\1\nrwl-go\node_modules\@angular-devkit\schematics\src\rules\call.js:78:24)
    at C:\Users\z0043rkh\Desktop\CP-11-NRWL\1\nrwl-go\node_modules\@angular-devkit\schematics\src\rules\call.js:73:40
    at Observable._subscribe (C:\Users\z0043rkh\Desktop\CP-11-NRWL\1\nrwl-go\node_modules\@angular-devkit\schematics\node_modules\rxjs\internal\observable\defer.js:10:21)
    at Observable._trySubscribe (C:\Users\z0043rkh\Desktop\CP-11-NRWL\1\nrwl-go\node_modules\@angular-devkit\schematics\node_modules\rxjs\internal\Observable.js:44:25)
    at Observable.subscribe (C:\Users\z0043rkh\Desktop\CP-11-NRWL\1\nrwl-go\node_modules\@angular-devkit\schematics\node_modules\rxjs\internal\Observable.js:30:22)
    at MapOperator.call (C:\Users\z0043rkh\Desktop\CP-11-NRWL\1\nrwl-go\node_modules\@angular-devkit\schematics\node_modules\rxjs\internal\operators\map.js:32:23)
le.js:25:31)
    at Object.innerSubscribe (C:\Users\z0043rkh\Desktop\CP-11-NRWL\1\nrwl-go\node_modules\@angular-devkit\schematics\node_modules\rxjs\internal\innerSubscribe.js:102:23)
    at MergeMapSubscriber._innerSub (C:\Users\z0043rkh\Desktop\CP-11-NRWL\1\nrwl-go\node_modules\@angular-devkit\schematics\node_modules\rxjs\internal\operators\mergeMap.js:80:50)
    at MergeMapSubscriber._tryNext (C:\Users\z0043rkh\Desktop\CP-11-NRWL\1\nrwl-go\node_modules\@angular-devkit\schematics\node_modules\rxjs\internal\operators\mergeMap.js:74:14)
Command failed: npx nx _migrate --run-migrations --verbose

 >  NX   Command failed: npx nx migrate --run-migrations --verbose

Error: Command failed: npx nx migrate --run-migrations --verbose
    at checkExecSyncError (node:child_process:828:11)
    at execSync (node:child_process:902:15)
    at runNxSync (C:\Users\z0043rkh\AppData\Local\Temp\tmp-34604-DeNlYPmbw8e9\node_modules\nx\src\utils\child-process.js:26:34)
    at C:\Users\z0043rkh\AppData\Local\Temp\tmp-34604-DeNlYPmbw8e9\node_modules\nx\src\command-line\migrate.js:935:43
    at Generator.next (<anonymous>)
    at C:\Users\z0043rkh\AppData\Local\Temp\tmp-34604-DeNlYPmbw8e9\node_modules\tslib\tslib.js:167:75
    at new Promise (<anonymous>)
    at Object.__awaiter (C:\Users\z0043rkh\AppData\Local\Temp\tmp-34604-DeNlYPmbw8e9\node_modules\tslib\tslib.js:163:16)
    at runMigrations (C:\Users\z0043rkh\AppData\Local\Temp\tmp-34604-DeNlYPmbw8e9\node_modules\nx\src\command-line\migrate.js:928:20)
    at Object.<anonymous> (C:\Users\z0043rkh\AppData\Local\Temp\tmp-34604-DeNlYPmbw8e9\node_modules\nx\src\command-line\migrate.js:1046:23)
Command failed: C:\Users\z0043rkh\AppData\Local\Temp\tmp-34604-DeNlYPmbw8e9\node_modules\.bin\nx _migrate --run-migrations --verbose

### Expected Behavior

To execute the migration change within the migrations.json file.

{
      "cli": "ng",
      "version": "15.2.0-beta.0",
      "requires": {
        "@angular/core": ">=15.0.0"
      },
      "description": "Remove exported `@angular/platform-server` `renderModule` method. The `renderModule` method is now exported by the Angular CLI.",
      "factory": "./src/migrations/update-15-2-0/remove-platform-server-exports",
      "package": "@nrwl/angular",
      "name": "update-platform-server-exports"
    },

### Steps to Reproduce

1.

### Nx Report

```shell
>  NX   Report complete - copy this into the issue template

   Node : 16.14.0
   OS   : win32 x64
   npm  : 8.3.1

   nx                      : 15.9.2
   @nrwl/js                : 15.9.2
   @nrwl/jest              : 15.9.2
   @nrwl/linter            : 15.9.2
   @nrwl/workspace         : 15.9.2
   @nrwl/angular           : 15.9.2
   @nrwl/cli               : 15.9.2
   @nrwl/cypress           : 15.9.2
   @nrwl/devkit            : 15.9.2
   @nrwl/eslint-plugin-nx  : 15.9.2
   @nrwl/nx-plugin         : 15.9.2
   @nrwl/tao               : 15.9.2
   @nrwl/webpack           : 15.9.2
   typescript              : 4.9.5
   ---------------------------------------
   Community plugins:
   @ngrx/effects        : 15.3.0
   @ngrx/entity         : 15.3.0
   @ngrx/router-store   : 15.3.0
   @ngrx/store          : 15.3.0
   ngx-spinner          : 15.0.1
   @compodoc/compodoc   : 1.1.19
   @ngrx/schematics     : 15.3.0
   @ngrx/store-devtools : 15.3.0
```

**Comment by leosvelperez** [comment]
This was fixed in https://github.com/nrwl/nx/pull/16157 and a patch version should be released soon with it. Please keep an eye on the releases to get the fix.

**Comment by bstrob** [comment]
@leosvelperez Will there be a patch version (e.g. 15.9.3) available with this fix? Or is it just via the 16.0.0-beta.0 available?

**Comment by jordi72** [comment]
This issue has been published? @leosvelperez
